src/neoxp/Commands/BatchCommand.BatchFileCommands.cs

```diff
@@ -57,7 +57,7 @@ internal class Deploy
                     internal string Password { get; init; } = string.Empty;
 
                     [Option(Description = "Witness Scope to use for transaction signer (Default: CalledByEntry)")]
-                    [AllowedValues(StringComparison.OrdinalIgnoreCase, "None", "CalledByEntry", "Global")]
+                    [McMaster.Extensions.CommandLineUtils.AllowedValues(StringComparison.OrdinalIgnoreCase, "None", "CalledByEntry", "Global")]
```

Comment:
Add to `using` statement `McMaster.Extensions.CommandLineUtils`?

Reply:
Already add, But `AllowedValues` has an unclear reference between `McMaster.Extensions.CommandLineUtils.AllowedValues` and `System.ComponentModel.DataAnnotations`.

Reply:
use `using AllowedValues = McMaster.Extensions.CommandLineUtils.AllowedValues`